web/pgadmin/tools/schema_diff/tests/utils.py

```
@@ -70,9 +70,58 @@ def restore_schema(server, db_name, schema_name, sql_path):
+    applied = []
+    pending = list(chunks)
+
+    while pending:
+        failed = []
+        for label, sql in pending:
+            try:
+                pg_cursor = connection.cursor()
+                pg_cursor.execute(sql)
+                pg_cursor.close()
+                applied.append(label)
+            except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)